### DISCUSSION TITLE
Welcome to schema_plus Discussions!

### DISCUSSION THREAD

**Discussion by ronen** [SchemaPlus/schema_plus#237] [General]
Hi @SchemaPlus/schema_plus-collaborators!  Are any of you still using schema plus? Any of you still able to support it?   

Myself it's been years since I've used RoR  -- haven't needed it for work nor had time to use it in a personal project.  So I have no idea what still works or doesn't in schema plus in the latest version of RoR.  Nor do I know what changes have been made to RoR that might obviate some of the features schema plus provides.  (I've always hoped that RoR would one day have the features making schema plus obsolete!  Maybe it does now??)

...So I haven't been responding to schema plus issues or maintaining it.

...And it seems like it's been years since anybody else has been doing so either.

So, is it time to "officially" declare schema plus abandonware? (I.e. add an appropriately-worded note to the README's.)

It'll be tough to do since I spent so many years working on it.  But I feel bad every time an issue gets filed without any response.

**Comment by lowjoel** [comment]
I find myself in the same position. I'd love to continue working on it but none of my work is in Rails today. I have several other gems in the same position too 😞

**Reply by tovodeverett** [reply]
I eliminated `schema_plus_pg_indexes` when I migrated to Rails 5.2, and I eliminated `schema_plus_foreign_keys` when I migrated to Rails 6.  One thing missing from Rails 6 that is in `schema_plus_foreign_keys` is support for deferrable constraints, but I worked around that by reworking my monkey patches for `yaml_db` to be Rails 6 foreign key aware and sequence the table population appropriately.

**Comment by ronen** [comment]
Well maybe it's not completely dead yet as per @urkle 's work on postgresql...  (thanks @urkle!)
